src/views/ComparisonView/ComparisonView.jsx

```diff
@@ -103,19 +102,16 @@ export function ComparisonView ({ companyComparison, peersComparison, isLoading,
   }
 
   return (
-    <>
-      <CardKPI title={'Peer Group Analysis'} actions={false} height={'80vh'} fullScreen={true}
-      topActions={
-        <ExportOption
-          buttonClass={classes.exportButton}
-          saveComparisonReport={saveComparisonReport}
-          isLoading={isLoading}
-          downloading={downloading}
-        />
-      }
-      >
+    <div style={{ overflow: 'none' }}>
```

Comment:
Can you add a condition here to display the comparison vs peers export data option?
This only works for comparison vs peers with the calendar year, not for investment year

Reply:
I have added the prop to the comparison view to evaluate when export data option should be displayed.